bin/start.sh

```diff
@@ -9,18 +9,23 @@ fi
 
 # prepare log path
 mkdir -p "$(dirname ${SERVER_LOG_PATH})"
+   
+# create a new log file with a timestamp
+timestamp=$(date +"%Y%m%d_%H%M%S")
+new_log_file="$(dirname ${SERVER_LOG_PATH})/server_${timestamp}.log"
+ln -sf "$new_log_file" "${SERVER_LOG_PATH}"  # Update the symbolic link to point to the new log file
 
 # startup python server
 project_root=$(dirname "$(pwd)")
 cd ${project_root} || exit 1
-nohup python ${project_root}/app/server/bootstrap.py &> ${SERVER_LOG_PATH} </dev/null &
+nohup python ${project_root}/app/server/bootstrap.py >> "${new_log_file}" 2>&1 </dev/null &
```

Comment:
if we create a new log file everytime in starting, maybe apending mode is not needed